packages/context/src/binding.ts

```diff
@@ -328,11 +349,26 @@ export class Binding<T = BoundValue> {
    */
   applyDefaultScope(scope: BindingScope): this {
     if (!this._scope) {
-      this._scope = scope;
+      this.inScope(scope);
     }
     return this;
   }
 
+  /**
+   * Set the `_getValue` function
+   * @param getValueFn getValue function
+   */
+  private _setGetValue(
```

Comment:
I find this name difficult to understand. How about the following?

```suggestion
  private _setValueGetter(
```